openedx/core/djangoapps/video_pipeline/management/commands/tests/test_create_video_pipeline_integration.py

```diff
@@ -0,0 +1,75 @@
+"""
+Tests of the create_oauth_client management command.
+"""
+
+import ddt
+from django.core.management import call_command
+from django.test import TestCase
+from openedx.core.djangoapps.video_pipeline.models import VideoPipelineIntegration
+
+
+@ddt.ddt
+class CreateVideoPipelineIntegration(TestCase):
+    """
+    Management command test class.
+    """
+    def setUp(self):
+        super(CreateVideoPipelineIntegration, self).setUp()
+
+    def assert_integration_created(self, args, options):
+        """
+        Verify that the integration was created.
+        """
+        integration = VideoPipelineIntegration.current()
+
+        for index, attr in enumerate(('client_name', 'api_url', 'service_username')):
+            self.assertEqual(args[index], getattr(integration, attr))
+
+        self.assertEqual(integration.enabled, options.get('enabled'))
+
+    @ddt.data(
+        (
+            [
+                'veda',
+                'http://veda.edx.org/api/',
+                'veda_service_user',
+            ],
+            {'enabled': False}
+        ),
+        (
+            [
+                'veda',
+                'http://veda.edx.org/api/',
+                'veda_service_user',
+            ],
+            {'enabled': True}
+        ),
+    )
+    @ddt.unpack
+    def test_integration_creation(self, args, options):
+        """
+        Verify that the command creates integration when given valid arguments and options.
```

Comment:
What about "Verify that the `create_video_pipeline_integration` command works as expected." ?